cloudigrade/instance/apps.py

```diff
@@ -0,0 +1,5 @@
+from django.apps import AppConfig
+
+
+class InstanceConfig(AppConfig):
+    name = 'instance'
```

Comment:
I'm not 100% on if we need an app for instance functionality, as I don't expect us to have too much instance specific functions. We should talk more about this.

Reply:
Quick summary of our conversation: Roll instance functionality into account app. General plan is to have three main apps, account, scheduler, and analyzer (well, util too). Account handles basic account setup and prep, scheduler deals with queues and scans, and analyzer actually processes the log files.